lib/crypto/index.js

```diff
@@ -872,13 +873,21 @@ Crypto.prototype.decryptEvent = function(event) {
         payload.keysProved = r.keysProved;
         return payload;
     } else {
-        // We've got a message for a session we don't have.
-        // Maybe the sender forgot to tell us about the session.
-        // Remind the sender that we exist so that they might
-        // tell us about the sender.
-        if (event.getRoomId !== undefined && event.getSender !== undefined) {
+        // We've got a message for a session we don't have.  Maybe the sender
+        // forgot to tell us about the session.  Remind the sender that we
+        // exist so that they might tell us about the session on their next
+        // send.
+        //
+        // (Alternatively, it might be that we are just looking at
+        // scrollback... at least we rate-limit the m.new_device events :/)
+        //
+        // XXX: this is a band-aid which masks symptoms of other bugs. It would
```

Comment:
uh, mayyybee? I'm afraid I'm not *entirely* clear on the difference between 1.2 and 1.3; and in any case I don't really think this is a good way to solve that problem.